addons/sale/models/sale_order.py

```diff
@@ -634,7 +639,13 @@ def _compute_invoice_status(self):
                 ['order_id', 'invoice_status']
             )
         ]
-        for order in confirmed_orders:
+
+        manually_closed_orders = self.filtered(
+            lambda o: o.is_manually_closed and o.state == 'sale'
+        )
```

Comment:
```suggestion
        manually_closed_orders = confirmed_orders.filtered(lambda o: o.is_manually_closed)
```
Since we already filtered confirmed orders